pkg/agent/cniserver/ipam/antrea_ipam.go

```diff
@@ -59,12 +60,17 @@ const (
 // Resource needs to be unique since it is used as identifier in Del.
 // Therefore Container ID is used, while Pod/Namespace are shown for visibility.
 // TODO: Consider multi-interface case
```

Comment:
This TODO can be removed I think?

Reply:
Make sense. Let me remove it in the next PR.